tests/functional/resolvers.ts

```diff
@@ -104,6 +104,7 @@ describe("Resolvers", () => {
           @Arg("stringArrayArg", type => String) stringArrayArg: string[],
           @Arg("explicitArrayArg", type => [String]) explicitArrayArg: any,
           @Arg("nullableStringArg", { nullable: true }) nullableStringArg?: string,
+          @Arg("defaultStringArg", { defaultValue: "bob" }) defaultStringArg?: string,
```

Comment:
Please try to follow the descriptive names convention, like `defaultStringArgDefaultValue` instead of `foo`, `bar`, `bob` and other sample names that might generate conflicts.